library/ssl_tls.c

```diff
@@ -6600,64 +6600,89 @@ int mbedtls_ssl_set_calc_verify_md(mbedtls_ssl_context *ssl, int md)
     return 0;
 }
 
-#if defined(MBEDTLS_MD_CAN_SHA256)
-int ssl_calc_verify_tls_sha256(const mbedtls_ssl_context *ssl,
-                               unsigned char *hash,
-                               size_t *hlen)
-{
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
-    size_t hash_size;
+static int ssl_calc_verify_tls_psa(const mbedtls_ssl_context *ssl,
+                                   const psa_hash_operation_t *hs_op,
+                                   size_t buffer_size,
+                                   unsigned char *hash,
+                                   size_t *hlen)
+{
     psa_status_t status;
-    psa_hash_operation_t sha256_psa = psa_hash_operation_init();
+    psa_hash_operation_t copy = psa_hash_operation_init();
```

Comment:
Good question, I wasn't sure. Since we have `hs_op` (resp. `hs_ctx` in the non-PSA case), it would probably make sense for this to be called `local_op` or `cloned_op` (resp. `local_ctx` or `cloned_ctx`) to better highlight the relationship between them.